prow/config/config.go

```diff
@@ -41,6 +43,8 @@ import (
 
 // Config is a read-only snapshot of the config.
 type Config struct {
+	Prow
```

Comment:
nit: We should put the presets and lists of jobs into an embedded struct, `Jobs`, similar to `Prow`.
Then we can define `mergeJobs` instead of using `mergeConfig` (which really just merges the jobs).

Reply:
ack